### DISCUSSION TITLE
K1 max error code 3002 whith gitbackup

### DISCUSSION THREAD

**Discussion by Ackta** [Guilouz/Creality-Helper-Script-Wiki#425] [Issues]
Hi,i`ve a key 3002 when i install git backup.
i must reset from factory to re run srcipt helper, and if i remove mooraker companion, the info system tell me i removed, but if i try the install  helper alert me companion is already installed.
last firmware from creality  installed.

**Comment by Guilouz** [comment]
Can you give me the correct issue you have in Moonraker ? 3002 is a generic error.

**Reply by Ackta** [reply]
on moonraker log ihave key 3002

the printer mus to be rese to default parameterafter this error.

**Reply by Guilouz** [reply]
3002 doesn't mean anything, I need the exact error

**Reply by Ackta** [reply]
ok but  i can make a backup before try to install git backup?
Another issue,  in the bed mesh leveling, if i inser any number in probe the result is a  6x6 probe count.it`my error , or a bug?
i`ve try to inser  a number only ex 8, or 8,8, or 8x8, but the result are the same  a 36 probing count.
